spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -76,7 +120,8 @@ def reset(self):
         # Reset the policies and environments via the controllers
         raise NotImplementedError
 
-    def nav(self, nav_target: str = None) -> Tuple[bool, str]:
+    @multimethod  # type: ignore
+    def nav(self, nav_target: str = None) -> Tuple[bool, str]:  # type: ignore
         """
         Perform the nav action on the specified navigation target
```

Comment:
Add "specified as a known string" at the end

Reply:
Addressed